dind/daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: kube-actions
+  name: binfmt-misc
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kube-actions
+      app.kubernetes.io/component: binfmt-misc
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kube-actions
+        app.kubernetes.io/component: binfmt-misc
+    spec:
+      initContainers:
+        - name: register
+          image: multiarch/qemu-user-static
+          args:
+            - --reset
+            - -p
+            - 'yes'
```

Comment:
`--persistent` makes it clearer. :smile:

Reply:
I'll change that!

Reply:
```suggestion
            - --persistent
            - 'yes'
```